AirCasting/SearchAndFollow/SearchLocation/SearchView.swift

```diff
@@ -20,32 +20,35 @@ struct SearchView: View {
         VStack(alignment: .leading) {
             title
             textField
-            parametersQuestion
-            HStack(spacing: 12) {
-                ForEach(viewModel.measurementTypes) { param in
-                    Button {
-                        withAnimation(.easeInOut(duration: 0.3)) {
-                            viewModel.onParameterTap(with: param.name)
-                        }
-                    } label: {
-                        Text(param.name)
-                            .padding(5)
-                    }.buttonStyle(MultiSelectButtonStyle(isSelected: param.isSelected))
-                        .padding(.bottom, 5)
-                }
-            }
-            sensorQuestion
-            HStack(spacing: 12) {
-                ForEach(viewModel.sensorTypes) { sensor in
-                    Button {
-                        viewModel.onSensorTap(with: sensor.name)
-                    } label: {
-                        Text(sensor.name)
-                            .padding(5)
-                    }.buttonStyle(MultiSelectButtonStyle(isSelected: sensor.isSelected))
-                        .padding(.bottom, 5)
-                }
-            }
+            #warning("This part was intentionally disabled, refer to https://trello.com/c/WqkeHh2x - to know more")
```

Comment:
Could you please add short explanation? You can quote Michael "openAQ and PurpleAir have revoked free access to their data via their APIs" :) It's a quicker check than going to trello, but you can leave the link, too